### ISSUE TITLE
Easier readability with same scale/unit per column

### ISSUE THREAD

**Issue by ottok** [allinurl/goaccess#178]
I just did a pull request #177 for right-aligning numbers in cells. This way it is easy to scan a column and spot values that are unusually big or small.

It would be even better if the scale/unit per column would be unified. For example if a time column has both microseconds and milliseconds, it is difficult to spot the slow pages because I need to read both the value and the unit. When it comes to http requests, I think it would be best to always show all timing values in milliseconds.

For sizes I'd show all values as KiB. If you don't unify for the whole binary, then at least please at least use the same scale inside the same column.

Thanks for a great program!

**Comment by ottok** [comment]
I am happy to implement this once me previous PR is merged.

I looked at the code and I think I can do a fairly clean implementation by adding new configuration options --fixed-bytes and --fixed-time which users can use to customize the size (GB, MB, KB etc) or time (mn, s, ms etc). Then I'd inject into util.c functions filesize_str and usecs_to_str checks so that the new config options (if set) are honoured.

Nice that your code is so well laid out and easy to read!

**Comment by allinurl** [comment]
@ottok Thanks for posting this. Since v1.0 is bringing a lot of changes, I'll ask you to please wait until this release is out, then I'll be happy to merge it.

**Comment by smeijer** [comment]
Are there any plans to implement this? Having KiB, MiB and GiB in the same column is very confusing.

**Comment by allinurl** [comment]
@smeijer Yes, there are plans. Unfortunately I haven't had the chance to get to it, but I do expect to have this implemented.

**Comment by dontcrash** [comment]
Any chance on having this? 👍
